soundfile.py

```diff
@@ -711,7 +733,8 @@ def __repr__(self):
         return ("SoundFile({0.name!r}, mode={0.mode!r}, "
                 "samplerate={0.samplerate}, channels={0.channels}, "
                 "format={0.format!r}, subtype={0.subtype!r}, "
-                "endian={0.endian!r})".format(self))
+                "endian={0.endian!r}, compression_level={0.compression_level}, "
```

Comment:
compression level and bitrate only make sense for compressed audio formats. They should not be printed for uncompressed formats.